### PR TITLE
A few C# fixes

### PR DESCRIPTION
**Pull request by tvandijck**

### CODE REVIEW

File: src/actions/vstudio/vs2010_vcxproj.lua [hunk 1]
```diff
@@ -1643,13 +1680,14 @@
 				isMakefile = true
 			end
 		end
-
```
Comment (samsinsane) [inline src/actions/vstudio/vs2010_vcxproj.lua]: This line should probably have stayed.

File: src/actions/vstudio/vs2010_vcxproj.lua [hunk 2]
```diff
@@ -1643,13 +1680,14 @@
...
 		if isWin then
 			if isMakefile then
 				m.element("Keyword", nil, "MakeFileProj")
 			else
-				if isManaged then
+				if isManaged or m.isClrMixed(prj) then
 					m.targetFramework(prj)
+				end
```
Comment (samsinsane) [inline src/actions/vstudio/vs2010_vcxproj.lua]: Should probably be an empty line here.

File: src/tools/dotnet.lua [hunk 3]
```diff
@@ -36,7 +36,6 @@
 
 		local fname = fcfg.abspath
 		local ext = path.getextension(fname):lower()
-
```
Comment (samsinsane) [inline src/tools/dotnet.lua]: This line should probably have stayed.

File: src/tools/dotnet.lua [hunk 4]
```diff
@@ -50,7 +49,11 @@
 			info.action = "Resource"
 		elseif ext == ".xaml" then
 			if fcfg.buildaction == "Application" or path.getbasename(fname) == "App" then
-				info.action = "ApplicationDefinition"
+				if fcfg.project.kind ~= premake.SHAREDLIB then
```
Comment (samsinsane) [inline src/tools/dotnet.lua]: Personal opinion: This would make more sense inversed since there's an else and this is a `~=`.

**Comment by tvandijck** [comment]
I'll fix the whitespace @samsinsane thanx for noticing...

**Pull request merged**